src/pages/library/dataBrowser-utils.ts

```diff
@@ -161,32 +151,12 @@ export const DatasetAccess = ({ dataset }: DatasetAccessProps) => {
   ])
 }
 
-// These are used to match against by the filter
-const extractTags = (dataset: DatasetResponse): Tags => {
-  return {
-    itemsType: 'AttributeValue',
-    items: _.flow(_.flatten, _.toLower)([
-      getDatasetAccessType(dataset),
-      getConsortiumTitlesFromDataset(dataset),
-      dataset.samples?.species,
-      dataset.samples?.disease,
-      getAssayCategoryListFromDataset(dataset),
-      getDataModalityListFromDataset(dataset),
-      _.map('TerraCore:hasFileFormat', dataset.fileAggregate),
-      dataset['TerraDCAT_ap:hasDataUsePermission']
-    ]) as string
-  }
-}
-
 
-export const prepareDatasetsForDisplay = (datasets: DatasetResponse[], dataCollectionsToInclude: string[]): Dataset[] => {
-  const filteredDatasets = _.filter(dataCollectionsToInclude ?
+export const prepareDatasetsForDisplay = (datasets: Dataset[], dataCollectionsToInclude: string[]): Dataset[] => {
+  return _.filter(dataCollectionsToInclude ?
```

Comment:
Can we avoid `return`ing here directly by doing `... => _.filter(dataCollectionsToInclude ? ...`